tests/univariate/test_KDEUnivariate.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `univariate` package."""
+
+from unittest import TestCase
+
+import numpy as np
+
+from copulas.univariate.KDEUnivariate import KDEUnivariate
+
+
+class TestKDEUnivariate(TestCase):
```

Comment:
This is just a minor styling issue but helps understanding the code: When possible, split each test in three blocks like:
```python
class TestKDEUnivariate(TestCase):

    def test_example(self):
        """On special circumstances, model may sample values."""
        # Setup
        model = KDEUnivariate()
        model.fit([1,2,3,4,5])

        # Run
        result = model.sample()

        # Check 
        assert result
```